*** ./smali/android/preference/RingtonePreference.smali	2015-11-28 22:11:13.498740310 +0900
--- /home/serenitys/patchrom-v6/f460/temp/new_smali/framework.jar.out/./smali/android/preference/RingtonePreference.smali	2015-11-28 22:09:31.566736768 +0900
***************
*** 311,316 ****
--- 311,322 ----
  
      invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->putExtra(Ljava/lang/String;Ljava/lang/CharSequence;)Landroid/content/Intent;
  
+     const-string v0, "com.android.thememanager"
+ 
+     const-string v1, "com.android.thememanager.activity.ThemeTabActivity"
+ 
+     invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->setClassName(Ljava/lang/String;Ljava/lang/String;)Landroid/content/Intent;
+ 
      return-void
  .end method
  
